modules/slider/main/index.coffee

```diff
@@ -62,9 +60,10 @@ slide = (e) ->
     _.dragging = 'value'
     val.value = posToValue.call this, pos
 
+  prevValue = @value()
   @value(val)
-
-  @emit 'change', @value()
+  if prevValue isnt @value()
+    @emit 'change', @value()
   return
```

Comment:
I guess so - although it's nice to explicitly return nothing rather than letting coffeescript return _something_, especially as I don't know if events use / change behaviour based on the return value